atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresDdlTableTest.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.keyvalue.dbkvs.impl.postgres;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Matchers.startsWith;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.google.common.collect.ImmutableMap;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.keyvalue.dbkvs.ImmutablePostgresDdlConfig;
+import com.palantir.atlasdb.keyvalue.dbkvs.impl.ConnectionSupplier;
+import com.palantir.atlasdb.keyvalue.dbkvs.impl.DbKvs;
+import com.palantir.nexus.db.DBType;
+import com.palantir.nexus.db.sql.AgnosticResultSetImpl;
+import com.palantir.nexus.db.sql.SqlConnection;
+
+public class PostgresDdlTableTest {
+    private static PostgresDdlTable postgresDdlTable;
+    private static final ConnectionSupplier connectionSupplier = mock(ConnectionSupplier.class);
+    private static final TableReference TEST_TABLE = TableReference.createFromFullyQualifiedName("ns.test");
+    private static final int NOW_MILLIS = 100;
+
+    @Before
+    public void setUp() {
+        postgresDdlTable = new PostgresDdlTable(TEST_TABLE,
+                connectionSupplier,
+                ImmutablePostgresDdlConfig.builder().compactIntervalMillis(10).build());
```

Comment:
We are recreating the table-object on each test, but the `postgresDdlTable` is static - is this intended?

Reply:
Nopes, not intended.